hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -827,79 +927,52 @@ logic is powered by VCAON.
 ## Core Power Control and Indication
 
 VCMAIN is the only supply that can be directly influenced by OpenTitan.
-The power manager can request VCMAIN to shutdown through main\_pd\_n.
-The state of VCMAIN is reflected by the vcmain\_pok\_o signal.
+The power manager can request VCMAIN to shutdown through main_pd_ni. The
+state of VCMAIN is reflected by the vcmain_pok_o signal.
 
 ## IO Power Indication
 
-IO power state is reflected to OpenTitan by vioa\_pok\_o and
-viob\_pok\_o signals
+IO power state is reflected to OpenTitan by vioa_pok_o and viob_pok_o

```

Comment:
it looks like below you are using backticks to render signal names with a code font.
You may want to do the same here as well: `vioa_pok_o`.